share/ansible_navigator/utils/key_value_store.py

```diff
@@ -18,74 +17,120 @@ def __init__(self, filename):
         cursor.execute("CREATE TABLE IF NOT EXISTS kv (key text unique, value text)")
 
     def close(self):
-        """close the connections"""
+        """Close the connections to the database."""
         self.conn.commit()
         self.conn.close()
 
     def open(self):
-        """establish the connection"""
+        """Establish the connection to the database."""
         self.conn = sqlite3.connect(self.path)
 
     def __len__(self):
+        """Determine the number of keys in the key-value store.
+
+        :returns: The number of keys
+        """
         cursor = self.conn.cursor()
         rows = cursor.execute("SELECT COUNT(*) FROM kv").fetchone()[0]
         return rows if rows is not None else 0
 
     def iterkeys(self):
-        """iterate keys"""
+        """Iterate the keys in the key-value store.
+
+        :yields: The keys in the key-value store
+        """
         cursor = self.conn.cursor()
         for row in cursor.execute("SELECT key FROM kv"):
             yield row[0]
 
     def itervalues(self):
-        """iterate values"""
+        """Iterate the values in the key-value store.
```

Comment:
I usually use `Emit` in generators, it seems to be the most accurate description of what `yield` does. @cidrblock WDYT?

Reply:
Emit, is a great word to lead the description of a yeild.